### PR TITLE
Cleanup of download-xorg-7.5

### PR DESCRIPTION
**Pull request by hackonteur**
Just fixing the issues I encountered in trying to build on RHEL6:
* wrong utf encoding for GNU Emacs
* Bad Mesa URL not handled well

The hardcoded Mesa URL is still broken, but I don't know which version to get instead.

    sh ../contrib/xorg/build-xorg init

### CODE REVIEW

**Comment by CendioOssman** [comment]
Looks good, but the commit messages aren't really properly formatted for git. You need to have a single line with a summary, followed by a blank line and then a more complete description (if necessary). This page is very good in describing what should go in a commit message:

http://chris.beams.io/posts/git-commit/

File: contrib/xorg/download-xorg-7.5 [hunk 1]
```diff
@@ -79,7 +113,8 @@ def main():
         else :
             fname = pkg + ".tar.gz"
         if not os.path.exists(fname):
-            assert 0 == os.spawnvp(os.P_WAIT, "wget", ["-N", "-c", "-O", fname, loc])
+            webget(loc, fname)
+            #assert 0 == os.spawnvp(os.P_WAIT, "wget", ["-N", "-c", "-O", fname, loc])
```
Comment (CendioOssman) [inline contrib/xorg/download-xorg-7.5]: Don't bother keeping dead code. Just delete this line.

**Comment by hackonteur** [comment]
Thanks for taking the time! I hadn't see the commit message guidelines. Dead code was a mistake. I'll try to clean up and make a new pull request.

**Comment by CendioOssman** [comment]
FYI, a new pull request is not necessary. Simply update the branch that is referenced in your pull request (which might require `--force` to overwrite the old commits).

Also, using named topic branches rather than `master` usually makes things more sane. :)

**Pull request closed without merging**